pkg/kvstore/etcd.go

```diff
@@ -433,6 +433,7 @@ func (e *etcdClient) renewSession() error {
 		e.UnlockIgnoreTime()
 		return fmt.Errorf("unable to renew etcd session: %s", err)
 	}
+	log.Infof("Got new lease ID %x", newSession.Lease())
```

Comment:
Approximately how often do we expect to receive a new lease ID? I'm wondering how many logs this will emit.

Reply:
As frequent as we lose connectivity with etcd or the user revokes the lease manually.